src/libcore/hash/sip.rs

```diff
@@ -19,15 +19,18 @@ use super::Hasher;
 ///
 /// See: http://131002.net/siphash/
 ///
-/// Consider this as a main "general-purpose" hash for all hashtables: it
-/// runs at good speed (competitive with spooky and city) and permits
-/// strong _keyed_ hashing. Key your hashtables from a strong RNG,
-/// such as `rand::Rng`.
+/// This is the default hashing function used by standard library (eg.
+/// [`collections::HashMap`](../collections/struct.HashMap.html) uses it by
+/// default).
```

Comment:
@bluss: AFAIK it will work just fine on [doc.rust-lang.org](https://doc.rust-lang.org/), which is where I believe the majority of users view these docs.